list-of-exercises/README.md

```diff
@@ -0,0 +1,17 @@
+---
+description: >-
+  In this section of the GitBook you will find a list of exercises that can be used in design sprint workshops.
+---
+
+# The List of Exercises
+
+Each of these exercises are adaptable and can be reorganised depending on the needs of the group you are facilitating. Tailoring the workshop by cherry picking specific exercises and arranging them in a certain order is a really powerful way of helping your user group process their problem areas and solutions. Your workshops become bespoke but still follow the design sprint structure.
```

Comment:
Could we remove reference to facilitating? Iirc, this page may be read by charities too?

Reply:
Yes!

Reply:
This isn't complete as I was just focusing on the structure so we can fix content later.